### DISCUSSION TITLE
Linux & Windows support

### DISCUSSION THREAD

**Discussion by kimmy-wang** [firebase/flutterfire#9512] [Feature request]
Linux & Windows support

**Comment by wiedymi** [comment]
Please, add support for Linux & Windows. It will be the best!

**Comment by malkhan-singh-gaur** [comment]
Please add windows and Linux support.🥺

**Comment by marb2000** [comment]
I do recommend to use the already opened discussion https://github.com/firebase/flutterfire/discussions/5557 and vote here.

**Comment by MichaelVerdon** [comment]
Closing as duplciate ticket https://github.com/firebase/flutterfire/discussions/5557
